### PR TITLE
fix: Update cli command

### PR DESCRIPTION
**Pull request by bensonarafat**
bug in git cli commend

### CODE REVIEW

File: src/pages/deploy-previews.astro [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Layout from '../components/layout.astro';
   description="See and share your changes in a production environment so you can ship with confidence. Gather feedback in your preferred project management tool right through your Deploy Previews!"
 >
   <main>
-    <h2>Try out Deploy Previews.</h2>
+    <h2>This is a Deploy Preview</h2>
```
Comment (rstavchansky) [inline src/pages/deploy-previews.astro]: Can you revert the change to this `deploy-previews.astro` file? We appreciate the fix to the repo name in the README, and we can merge that in once you remove this edit.

**Review by rstavchansky — CHANGES_REQUESTED** [review]
Thanks for this PR. Once you revert your update to the `deploy-previews.astro` file, I can accept the PR's update to the README file instructions.

**Comment by bensonarafat** [comment]
Okay now.

**Review by rstavchansky — APPROVED** [review]
Please go ahead and merge this pull request. Thank you for your contribution!

**Pull request closed without merging**